libflux/flux-core/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-#![cfg_attr(feature = "strict", deny(warnings, missing_docs))]
```

Comment:
Why did we remove this line? I see it's added elsewhere, but I'm pretty sure we want this line here in the top `lib.rs`.

Reply:
I must have missed reverting this.

Reply:
I saw that this line was removed and I readded it to the wrong place.